zh/backup-to-aws-s3-using-br.md

```diff
@@ -342,6 +349,10 @@ kubectl get bk -n test1 -o wide
 
     如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
```

Comment:
Please help update the `tikv_gc_lifetime` to `tikv_gc_life_time` in the docs, thanks!